### PR TITLE
Livechat

### PR DESCRIPTION
**Pull request by tinyfishbigpond**

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -53,9 +53,16 @@ const App = () => {
   return (
     <AuthProvider>
       <div>
+<<<<<<< HEAD
+        {/* Home Page */}
+        <Modal classname='mfn_modal'show={modalState}/>
```
Comment (Symphon-y) [inline client/src/App.jsx]: Modal and mfn button should be commented out, these are meant to be imported in individual components

**Review by Symphon-y — CHANGES_REQUESTED** [review]
Modal and mfn button should be commented out, these are meant to be imported in individual components

**Review by sharonhw888 — APPROVED** [review]
ok

File: client/src/App.jsx (hunk 1 above)
Comment (tinyfishbigpond) [reply client/src/App.jsx]: Got it I commented them out

**Review by Symphon-y — APPROVED** [review]

**Pull request merged**